external/boost/archive/portable_binary_oarchive.hpp

```diff
@@ -171,7 +171,7 @@ class portable_binary_oarchive :
 
     void init(unsigned int flags);
 public:
-    portable_binary_oarchive(std::ostream & os, unsigned flags = 0) :
+    portable_binary_oarchive(std::ostream & os, unsigned flags = endian_little) :

```

Comment:
Does anyone use the flags feature currently? I think this argument should be dropped here and in the other constructor below. Instead, force the `endian_little` flag.

This change here still allows someone to _not_ set either endianess when storing.

Reply:
I don't think anything uses it now.